### PR TITLE
logfmt support

### PR DESCRIPTION
**Pull request by maximmi**
## what
* fluentd config to parse `logfmt` logs from kubernetes application 
* gemfiles required for fluentd config

# why
* to be able to parse logs from application in `logfmt` format and pass it to ElasticSearch in nice way

### CODE REVIEW

**Review by Nuru — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.build-harness
+build-harness/
+aws-assumed-role/
```
Comment (aknysh) [inline .gitignore]: remove `aws-assumed-role/`

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Canonical GitHub repo
+github_repo: cloudposse/fluentd-kubernetes-logformat
+
+# Logo for this project
+#logo: docs/logo.png
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/prometheus-to-cloudwatch.svg?branch=master"
```
Comment (aknysh) [inline README.yaml]:
```suggestion
    image: "https://travis-ci.org/cloudposse/fluentd-kubernetes-logformat.svg?branch=master"
```

File: README.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    url: "https://travis-ci.org/cloudposse/prometheus-to-cloudwatch"
```
Comment (aknysh) [inline README.yaml]:
```suggestion
    url: "https://travis-ci.org/cloudposse/fluentd-kubernetes-logformat"
```

File: README.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    image: "https://img.shields.io/github/release/cloudposse/fluentd-kubernetes-logformat.svg"
+    url: "https://github.com/cloudposse/fluentd-kubernetes-logformat/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+
+  - name: "Fluentd Daemonset for Kubernetes"
+    description: "Fluentd daemonset for Kubernetes and it Docker image"
```
Comment (aknysh) [inline README.yaml]:
```suggestion
    description: "Fluentd daemonset for Kubernetes and its Docker image"
```

File: README.yaml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    url: "https://github.com/cloudposse/fluentd-kubernetes-daemonset"
+
+description: |-
+
+  Adds parsing features for kubernetes logs corresponding to logformat pod label
+
+usage: |-
+
+  Add `logformat` label with log format of your application helm chart. Make sure `fluentd` supports it (`conf.d` folder)
```
Comment (aknysh) [inline README.yaml]:
```suggestion
  Add `logformat` label with log format of your application helm chart. Make sure `fluentd` supports it (see `conf.d` folder)
```

File: README.yaml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+examples: |-
+  ### Build Go program
```
Comment (aknysh) [inline README.yaml]: remove

File: README.yaml [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        logformat: "logfmt"
+    strategy:
+      type: "RollingUpdate"
+      rollingUpdate:
+        maxUnavailable: 1
+    revisionHistoryLimit: 5
+  ...
+  ```
+
+screenshots:
```
Comment (aknysh) [inline README.yaml:40]: remove screenshots or update the `name`, `url` and `description` if we have screenshots related to the repo

File: README.yaml [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+related:
+
+  - name: "Fluentd Daemonset for Kubernetes"
+    description: "Fluentd daemonset for Kubernetes and it Docker image"
+    url: "https://github.com/cloudposse/fluentd-kubernetes-daemonset"
+
+description: |-
+
+  Adds parsing features for kubernetes logs corresponding to logformat pod label
```
Comment (aknysh) [inline README.yaml:34]:
```suggestion
  Allows labeling Kubernetes pods with `logformat` labels to support different log formats
  Adds parsing features for kubernetes logs corresponding to logformat pod label
```

**Review by aknysh — DISMISSED** [review]
see comments

File: README.yaml (hunk 6 above)
Comment (maximmi) [reply README.yaml]: done

File: README.yaml (hunk 5 above)
Comment (maximmi) [reply README.yaml]: done

File: README.yaml (hunk 4 above)
Comment (maximmi) [reply README.yaml]: done

File: README.yaml (hunk 3 above)
Comment (maximmi) [reply README.yaml]: done

File: README.yaml (hunk 2 above)
Comment (maximmi) [reply README.yaml]: done

File: .gitignore (hunk 1 above)
Comment (maximmi) [reply .gitignore]: done

**Review by goruha — APPROVED** [review]

**Pull request merged**